sw/device/sca/ecc256_sign_serial.c

```diff
@@ -96,7 +96,7 @@ uint32_t ecc256_msg[kEcc256NumWords] = {
 };
 
 // p256_ecdsa_sca has randomnization removed.
-OTBN_DECLARE_APP_SYMBOLS(p256_ecdsa_sca);
+OTBN_DECLARE_APP_SYMBOLS(p256_ecdsa_sca)
```

Comment:
can we update the macro, e.g. remove the last semicolon, for the sake of consistency?